reframe/schemas/runreport.json

```diff
@@ -27,25 +30,25 @@
             "job_stderr": {"type": ["string", "null"]},
             "maintainers": {
                 "type": "array",
-                "items": {"type": ["string", "null"]}
+                "items": {"type": ["string"]}
             },
             "tags": {
                 "type": "array",
-                "items": {"type": ["string", "null"]}
+                "items": {"type": ["string"]}
             },
-            "run_no": {"type": "number"},
-            "time_setup": {"type": "number"},
+            "runid": {"type": "number"},
+            "time_setup": {"type": ["number", "null"]},
             "time_compile": {"type": ["number", "null"]},
             "time_run": {"type": ["number", "null"]},
             "time_sanity": {"type": ["number", "null"]},
             "time_performance": {"type": ["number", "null"]},
-            "time_total": {"type": "number"}
+            "time_total": {"type": ["number", "null"]}
```

Comment:
@ekouts Can total be `None`??

Reply:
yes, when it fails because of a dependency it is `None`.

Reply:
Hmm, I think we should fix that in the runner. It doesn't make too much sense.

Reply:
It's because it gets the difference between the first and last timestamp:
https://github.com/eth-cscs/reframe/blob/master/reframe/frontend/executors/__init__.py#L144
Would you prefer to return 0 for `total` in that case? When it fails directly from a dependency what would we count as total time? The time in the runcase, since it was processed? Or the time that it was waiting?

Reply:
@ekouts We could get a timestamp at the time when the task is created.